envoy/api/v2/core/health_check.proto

```diff
@@ -124,6 +124,10 @@ message HealthCheck {
     // gRPC health check.
     GrpcHealthCheck grpc_health_check = 11;
   }
+
+  // [#not-implemented-hide:] The no traffic interval between health checks.
```

Comment:
This needs a better description or a link to somewhere that has it. As a user I can't tell what this refers to.

Reply:
+1. Sorry, this "feature" I think is basically not documented. If you need help documenting it please let me know or grab me on slack.

Reply:
@mattklein123 yes I need it. I'm not really sure what to put while I have the corresponding envoy changeset ready modulo this one. Could you help me to understand it better?

Reply:
The idea behind the NO_TRAFFIC_INTERVAL is to have a slower health check rate if a cluster is defined but has never made a connection. This originates from the very old days of Envoy in which we didn't have a lot of dynamic configuration at Lyft and thus we had lots of clusters defined that were never used. 

One option here is to just delete this functionality, but it's possible people are relying on it and don't realize it. I do think making it configurable makes sense if we keep it. The documentation could look something like:

The "no traffic interval" is a special health check interval that is used when a cluster has never had traffic routed to it. This lower interval allows cluster information to be kept up to date, without sending a potentially large amount of active health checking traffic for no reason. Once a cluster has been used for traffic routing, Envoy will shift back to using the standard health check interval that is defined.

Reply:
Thanks @mattklein123